packages/cadl-ranch-specs/http/client/structure/common/service.tsp

```diff
@@ -29,7 +29,6 @@ using Azure.ClientGenerator.Core;
 )
 @service({
   title: "MultiClient",
-  version: "1.0.0",
```

Comment:
we should add `@versioned` to avoid breaking ?

Reply:
this should not have been used and is the entire point why we deprecating `version:`

Reply:
oh because there's no api version parameter in it? got it, thanks

Reply:
yeah especially but either way